src/task.js

```diff
@@ -0,0 +1,68 @@
+export class Duty {
+  constructor(description) {
+    this.description = description;
+    this.completed = false;
+    this.index = 0;
+  }
+
+  push(arr) {
+    this.index = arr.length;
+    arr.push(this);
+    window.localStorage.setItem('tasklist', JSON.stringify(arr));
+    window.location.reload();
+  }
+}
+
+export function removecompleted(arr) {
+  const todelete = [];
+  arr.forEach((element) => {
+    if (element.completed === true) {
+      todelete.push(element);
+      const filteredArray = arr.filter((item) => !todelete.includes(item));
+      window.localStorage.setItem('tasklist', JSON.stringify(filteredArray));
+      window.location.reload();
+    }
+  });
+}
```

Comment:
- [ ] Well done implementing the remove completed tasks function 👏 However, there is still one more thing to consider: re-sorting the indexes of the array's list (tasks) when some tasks are deleted. For example, when you have 4 tasks,
```
one > id=1
two > id=2
three > id=3
four > id=4
```
Then we delete two and four
It should now be like
```
two > id=1
three > id=2
```
Currently, each task's id isn't changing, so please let's work on that to match the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique.)

_Take a look at the image for more clarification._
![Webpack Exercise - Google Chrome 7_24_2021 3_47_01 AM (4)](https://user-images.githubusercontent.com/47625407/126854127-7fc8e05f-5a38-4474-9f91-b97c964379bd.png)